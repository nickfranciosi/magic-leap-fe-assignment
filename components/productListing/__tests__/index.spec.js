/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import ProductListing from '../index';

describe('ProductListing', () => {
  test('it should render', () => {
    const tree = renderer
      .create(<ProductListing product={{ name: '123' }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
