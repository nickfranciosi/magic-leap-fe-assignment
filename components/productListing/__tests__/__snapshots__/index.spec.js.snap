// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductListing it should render 1`] = `
.c0 {
  border: #151515;
  width: 100%;
  height: auto;
  min-height: 250px;
  border: 1px solid #9e4f60;
  position: relative;
  cursor: pointer;
  background: url('/static/images/products/.jpg') no-repeat;
  background-size: cover;
  object-fit: cover;
}

.c2 {
  color: #fff;
  font-family: 'Space Mono',monospace;
  position: absolute;
  bottom: 24px;
  left: 6px;
  font-size: 32px;
}

.c1 {
  position: absolute;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  opacity: 0.67;
  z-index: 1;
  -webkit-transition: opacity 350ms ease;
  transition: opacity 350ms ease;
  background-image: linear-gradient( to right top,#000000,rgba(0,0,0,0) 56% );
}

.c1:hover {
  opacity: 0;
}

<div
  className="c0"
  onClick={[Function]}
  src={undefined}
>
  <div
    className="c1"
  />
  <h3
    className="c2"
  />
</div>
`;
